pkg/cli/bootstrap.go

```diff
@@ -0,0 +1,255 @@
+// Copyright 2021 Outreach Corporation. All Rights Reserved.
+
+// Description: This file contains cli functions used in bootstrap
+// and eventually in stencil.
+package cli
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"os/signal"
+	"os/user"
+	"runtime"
+	"runtime/debug"
+	"strings"
+	"syscall"
+
+	"github.com/getoutreach/gobox/pkg/app"
+	"github.com/getoutreach/gobox/pkg/cfg"
+	"github.com/getoutreach/gobox/pkg/exec"
+	"github.com/getoutreach/gobox/pkg/log"
+	"github.com/getoutreach/gobox/pkg/secrets"
+	"github.com/getoutreach/gobox/pkg/trace"
+	"github.com/getoutreach/gobox/pkg/updater"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	// UpdateExitCode is the exit code returned when an update ocurred
+	UpdateExitCode = 5
+)
+
+// overrideConfigLoaders fakes certain parts of the config that usually get pulled
+// in via mechanisms that don't make sense to use in CLIs.
+func overrideConfigLoaders(honeycombApiKey string, dataset string) {
+	var fallbackSecretLookup func(context.Context, string) ([]byte, error)
+	fallbackSecretLookup = secrets.SetDevLookup(func(ctx context.Context, key string) ([]byte, error) {
+		if key == "APIKey" {
+			return []byte(honeycombApiKey), nil
+		}
+
+		return fallbackSecretLookup(ctx, key)
+	})
+
+	log.SetOutput(io.Discard)
```

Comment:
is this just to make sure we're using logrus over gobox log

Reply:
yes